<launch>
<node_container pkg="rclcpp_components" exec="component_container_mt" name="ouagv_ekf_container" namespace="ouagv_ekf" args="" output="screen">
<composable_node name="ouagv_ekf_node" pkg="ouagv_ekf" plugin="ouagv_ekf::EkfComponent">
    <param name="reference_frame_id" value="map"/>
    <param name="base_frame_id" value="base_link"/>
    <param name="sigma_odom" value="0.00001"/>
    <param name="sigma_ndt_pose" value="0.1"/>
    <param name="sigma_imu" value="0.00001"/>
    <param name="use_imu" value="true"/>
    <param name="dt" value="0.05"/>
    <remap from="/odom" to="/diff_drive_base_controller/odom"/>
    <remap from="/imu" to="/imu/data"/>
</composable_node>
</node_container>
</launch>
